numpy/core/fromnumeric.py

```diff
@@ -3560,6 +3560,12 @@ def std(a, axis=None, dtype=None, out=None, ddof=0, keepdims=np._NoValue, *,
         See `~numpy.ufunc.reduce` for details.
 
         .. versionadded:: 1.20.0
+        
+    mean : optionally provide the mean to prevent recalculation, the mean
```

Comment:
Nit: I don't like long sentences. If you decide to go with my suggestion, please do it everywhere. Also, some of the docstrings do not have quotes around `keepdims = True`
```suggestion
    mean : optionally provide the mean to prevent recalculation. The mean
```

Reply:
The quotes are there now, also added description of the type and the label optional.